packages/components/psammead-play-button/README.md

```diff
@@ -0,0 +1,75 @@
+# ⛔️ This is an alpha component ⛔️
+
+This component is currently tagged as alpha and is not suitable for production use. Following the passing of an accessibility review this component will be marked as ready for production and the alpha tag removed.
+
+# psammead-play-button - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-play-button%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-play-button%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-play-button)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-play-button) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-play-button)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-play-button&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/play-button/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/play-button--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-play-button.svg)](https://www.npmjs.com/package/@bbc/psammead-play-button) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `PlayButton` component renders a button with a 'video' or 'audio' icon and optional duration timestamp. Which icon is displayed is defined by the value of the `type` prop. The component displays a 'video' icon by default. This component has options for displaying a duration timestamp, which if not provided will render gracefully without. When clicked, the function passed to `onClick` will be called.
+
+## Installation
+
+`npm install @bbc/psammead-play-button`
+
+## Props
+
+<!-- prettier-ignore -->
+| Argument       | Type      | Required | Default   | Example                  |
+| -------------- | --------- | -------- | --------- | ------------------------ |
+| service        | string    | Yes      | N/A       | `'news'`                 |
+| title          | string    | Yes      | N/A       | `'Foo.'`                 |
+| onClick        | function  | Yes      | N/A       | `() => {}`               |
+| duration       | string    | No       | `null`    | `'2:30'`                 |
+| durationSpoken | string    | No       | `null`    | `'2 minutes 30 seconds'` |
+| datetime       | string    | No       | `null`    | `'PT2M30S'`              |
+| type           | string    | No       | `'video'` | `'audio'`                |
+| className      | string    | No       | `null`    | `'bar'`                  |
```

Comment:
`className` is an ingrained part of React probably doesn't need to be mentioned as a prop here.

Reply:
I wouldn't normally put `className` in there, but this component needs to expect to be styled by its parent. In Media Player we use [this Styled Components functionality](https://www.styled-components.com/docs/basics#styling-any-component) to do that.

Reply:
I'm not sure what you mean here, we just seem to be adding the `className` to the `<Button>`. I also can't see anywhere in https://github.com/bbc/psammead/blob/latest/packages/components/psammead-media-player/src/Placeholder/index.jsx or https://github.com/bbc/simorgh/blob/latest/src/app/containers/MediaPlayer/index.jsx where we are styling via the parent component. 🤔

Reply:
In order to do:
```
import PlayButton from '@bbc/psammead-play-button';

const StyledPlayButton = styled(PlayButton)`
  position: absolute;
  bottom: 0;
`;
```
It's required to pass `className` to `className` - confusing, I know, but take a look at the Styled Components link. It has a good example.

This isn't in Placeholder yet, but will be.